src/contract.rs

```diff
@@ -329,6 +413,36 @@ pub mod tests {
         assert_eq!(10_000, coins.first().unwrap().amount.u128());
     }
 
+    #[test]
+    fn issue_restricted_coin_withdrawal() {
+        let mut deps = restricted_capital_coin_deps(None);
+        load_markers(&mut deps.querier);
+
+        let res = execute(
+            deps.as_mut(),
+            mock_env(),
+            mock_info("gp", &[]),
+            HandleMsg::IssueWithdrawal {
+                to: Addr::unchecked("omni"),
+                amount: 10_000,
+                memo: None,
+            },
+        )
+        .unwrap();
+
+        // verify that send message is sent
+        assert_eq!(1, res.messages.len());
+        println!("{:?}", msg_at_index(&res, 0));
```

Comment:
not sure we want to keep this in?

Reply:
Crap..... yep did not mean to keep that in, thanks